### PR TITLE
local changes from cob4-7

### PR DESCRIPTION
**Pull request by benmaidel**

### CODE REVIEW

File: cob_bringup/env.sh [hunk 1]
```diff
@@ -13,4 +13,6 @@ else
     source /opt/ros/indigo/setup.bash
 fi
 
+export ROS_IP=`hostname -I | awk '{print $1}'`
```
Comment (floweisshardt) [inline cob_bringup/env.sh:16]: @ipa-fxm and @ipa-nhg: do you see any troubles when adding this to our generic `env.sh` script?

File: cob_hardware_config/robots/cob4-7/config/pc_monitor_cob4-7-h1.yaml [hunk 2]
```diff
@@ -2,5 +2,5 @@ check_core_temps: False
 check_ipmi_tool: False
 enforce_clock_speed: False
 num_cores: 4
-load1_threshold: 6
-load5_threshold: 4
+load1_threshold: 8
```
Comment (floweisshardt) [inline cob_hardware_config/robots/cob4-7/config/pc_monitor_cob4-7-h1.yaml]: we should use the same values here as for cob4-2. I'll update this PR

**Review by floweisshardt — CHANGES_REQUESTED** [review]

File: cob_bringup/robots/cob4-7.xml [hunk 3]
```diff
@@ -44,7 +44,6 @@
 		</include>
 		<include file="$(find cob_bringup)/drivers/bms.launch">
 			<arg name="robot" value="$(arg robot)"/>
-			<arg name="sim" value="true"/>
```
Comment (fmessmer) [inline cob_bringup/robots/cob4-7.xml:47]: we need this line to be `<arg name="sim" value="$(arg sim)"/>` otherwise we'll break simulation!

**Review by fmessmer — CHANGES_REQUESTED** [review]

File: cob_bringup/robots/cob4-7.xml (hunk 3 above)
Comment (benmaidel) [reply cob_bringup/robots/cob4-7.xml:47]: agreed!

File: cob_bringup/env.sh (hunk 1 above)
Comment (fmessmer) [reply cob_bringup/env.sh:16]: what content is in `$1`?
I would like to add this to my the `.bashrc` on my laptop :wink:

File: cob_bringup/robots/cob4-7.xml (hunk 3 above)
Comment (floweisshardt) [reply cob_bringup/robots/cob4-7.xml:47]: done in 2147ca8096baa9fac945ec812ada0b67ff3a23a0

File: cob_bringup/env.sh (hunk 1 above)
Comment (benmaidel) [reply cob_bringup/env.sh:16]: `$1` is the first IP Adress which `hostname -I` prints

Comment (NadiaHG) [reply cob_bringup/env.sh:16]: In principle should be ok, was already tested on the robot? also sourcing the env.sh of other user to execute the robot.launch?

**Pull request merged**